WordPress/src/main/java/org/wordpress/android/util/analytics/AnalyticsUtils.java

```diff
@@ -110,8 +122,8 @@ public static void refreshMetadata(AccountStore accountStore, SiteStore siteStor
         if (siteLocalId != -1) {
             // Site previously selected, use it
             SiteModel selectedSite = siteStore.getSiteByLocalId(siteLocalId);
-            // If saved site exist, then add info
-            if (selectedSite != null) {
+            // If saved site exist, then add editor info
+            if (selectedSite != null && !TextUtils.isEmpty(selectedSite.getMobileEditor())) {
```

Comment:
Apparently ([see here](https://github.com/wordpress-mobile/WordPress-Android/blob/b3af4aeeaa176902280352ceb70860c7daae5bfa/libs/analytics/WordPressAnalytics/src/main/java/org/wordpress/android/analytics/AnalyticsTrackerNosara.java#L527) that the flag is set in the user section) we'll write `false` by default even if we skip setting the flag here. Perhaps leaving empty is a better approach so, wdyt about introducing an additional flag in `AnalyticsMetadata` to denote if we have set the flag at `AnalyticsMetadata.mIsGutenbergEnabled` at all or not?

Reply:
Addressed in b59bd9126709f8a11280f2105fc5398924c2fa02